src/Runner/Job.php

```diff
@@ -245,4 +245,24 @@ public function getHeaders()
 		return $this->headers;
 	}
 
+
+	/**
+	 * @return string
+	 */
+	private function buildArgumentsList()
+	{
+		$args = [];
+
+		foreach ($this->args as $key => $arg) {
```

Comment:
What do you mean by "--$key=$arg is not standard format"? It's the exact same format as the one [used in the current implementation](https://github.com/nette/tester/blob/7c21d99/src/Runner/TestHandler.php#L54)